backend/src/test/java/pl/edu/agh/niebieskiekotki/groups/GroupGeneratorTests.java

```diff
@@ -0,0 +1,73 @@
+package pl.edu.agh.niebieskiekotki.groups;
+
+import org.junit.jupiter.api.Test;
+import pl.edu.agh.niebieskiekotki.entitites.Student;
+import pl.edu.agh.niebieskiekotki.entitites.Term;
+import pl.edu.agh.niebieskiekotki.utility.Days;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GroupGeneratorTests {
+
+    GroupGenerator generator = new GroupGenerator();
+
+    @Test
+    public void test(){
+        Student s1 = new Student();
+        Student s2 = new Student();
+        Student s3 = new Student();
+        Student s4 = new Student();
+        Student s5 = new Student();
+        Student s6 = new Student();
+        Student s7 = new Student();
+        Student s8 = new Student();
+
+        Term t1 = new Term(1L, Days.Monday, 1, null);
+        Term t2 = new Term(2L, Days.Tuesday, 2, null);
+        Term t3 = new Term(3L, Days.Wednesday, 3, null);
+        Term t4 = new Term(4L, Days.Thursday, 4, null);
+        Term t5 = new Term(5L, Days.Friday, 5, null);
+        Term t6 = new Term(6L, Days.Monday, 6, null);
+        Term t7 = new Term(7L, Days.Thursday, 3, null);
```

Comment:
Trzeci argument to tylko zmienna określająca czy to jest grupa a/b albo inne dodatkowe informacje na temat terminu. W tym przypadku nie zmienia to nic w logice testowania, ale wolałem się upewnić że o tym wiesz i nie korzystasz nigdzie z tego że wartości tej zmiennej są różne, bo nie muszą być. (Nie widziałem takiego kodu, ale lepiej napisać niż nie)